### PR TITLE
next: Svelte 5

### PR DESCRIPTION
**Pull request by shyakadavis**

### CODE REVIEW

**Comment by shyakadavis** [comment]
Had forgotten about this one.
@ieedan any thoughts?

**Comment by ieedan** [comment]
Yeah I am looking right now!

image
Glad you got this one figured out it looks sick

**Comment by ieedan** [comment]
My work laptop seems to agree that this is a very large PR and it really doesn't like me trying to do anything though the website!

File: src/lib/components/ui/show-more/show-more.svelte [hunk 1]
```diff
@@ -3,12 +3,17 @@
...
+	};
+
+	let { expanded, onclick }: Props = $props();
 </script>
 
 <div class="relative flex min-h-[30px] w-[calc(100%-40px)] items-center p-5">
 	<div aria-hidden="true" class="h-px w-full -translate-x-px bg-gray-alpha-400"></div>
-	<Button variant="secondary" size="sm" shape="rounded" aria-label="Show less" on:click>
+	<Button variant="secondary" size="sm" shape="rounded" aria-label="Show less" {onclick}>
 		Show less
```
Comment (ieedan) [inline src/lib/components/ui/show-more/show-more.svelte:17]: Change this to:
```svelte
{#if expanded}
	Show Less
{:else}
	Show More
{/else}
```

File: src/lib/components/ui/show-more/show-more.svelte [hunk 2]
```diff
@@ -3,12 +3,17 @@
...
 		<Icons.ChevronLeft
 			class={cn('size-4 -rotate-90 transition-transform duration-200 ease-in-out', {
```
Comment (ieedan) [inline src/lib/components/ui/show-more/show-more.svelte:19]: This is a bit oversized compared to the original maybe `size-3` would work better.

**Comment by ieedan** [comment]
That's my quick review. Overall I think everything works pretty good with a few exceptions noted above! I didn't put every difference but instead just the glaringly obvious stuff I think everyone will notice or things that aren't the intended behavior.

**Comment by ieedan** [comment]
Also if you want people to be able to add these components kinda shadcn-style without having to copy the code themselves you can probably use the project I am working on called [jsrepo](https://github.com/ieedan/jsrepo) could be cool cause then it can be almost a drop in replacement for `shadcn-svelte` with the vercel styling.

**Comment by shyakadavis** [comment]
Thank you very much, @ieedan Caught a bunch of stuff I had missed.
I'm holding off on renaming the components just yet, but will eventually do.
Also, Vercel changed a bunch of stuff, for sure. Saw even new components since my last visit.

As for `jsrepo` been meaning to try it out (subbed to your channel last week 😁). We could also a note in this repo's readme. WDYT?

**Comment by ieedan** [comment]
Yeah I don't think we need to add anything on the site like `jsrepo add <component>` but a note in the README would work. Anyone using this will have to visit the GH anyways.

It does require (minimal) configuration so if you would prefer me to open a PR for it I can!

**Comment by shyakadavis** [comment]
>  if you would prefer me to open a PR for it I can!

Please do. Let merge this first.

**Comment by ieedan** [comment]
You forgot the progress bar lol

**Comment by shyakadavis** [comment]
> You forgot the progress bar lol

Snap. 😅 Will open a follow-up later tonight, if not, tomorrow. (Bout to head home right now)

**Comment by ieedan** [comment]
NP I should be able to open one for it!

**Pull request merged**